modules/expectations.lua

```diff
@@ -95,6 +96,11 @@ function Expectations.Expectation(name, connection)
 
   --- Perform base validation of expectation and set result into `Test`
   function mt.__index:validate()
+    if self.isAtLeastOneFail == true then
```

Comment:
@dboltovskyi isn't this an overhead to compare boolean values with true/false? Maybe it is enough just to write:
`if self.isAtLeastOneFail then`

Reply:
@AKalinich-Luxoft In this case - yes
But due to lua syntax there will be tricky moment in case ``` if not param then ... ```
In this case code will be executed in cases: 
 - if param is false
 - if param is nil

So I propose to define the condition explicitly